.github/workflows/TestEnv.yml

```
@@ -39,7 +40,10 @@ jobs:
           ddev composer config --no-plugins allow-plugins.lullabot/drainpipe true
           ddev composer config repositories.drainpipe --json "{\"type\": \"path\", \"url\": \"drainpipe\", \"options\": {\"symlink\": false}}"
           ddev composer config minimum-stability dev
-          cat composer.json | jq --indent 4 '."autoload-dev" = {"files": ["vendor/lullabot/drainpipe/scaffold/env/dotenv.php"]}' | tee composer.json > /dev/null
+          # cat + tee still has a race condition causing random failures.
+          # https://stackoverflow.com/a/68676400
+          cat composer.json | jq --indent 4 '."autoload-dev" = {"files": ["vendor/lullabot/drainpipe/scaffold/env/dotenv.php"]}' | tee composer.new.json
```

shellcheck recommends to remove the cat

```suggestion
          jq --indent 4 '."autoload-dev" = {"files": ["vendor/lullabot/drainpipe/scaffold/env/dotenv.php"]}' < composer.json > composer.new.json
```

---

Did it indicate an possible bug, or just using the shell instead of cat?

I would like to keep the `tee` call just because it helps to see what the generated file is in the build logs.

---

Just a warning…

This might also work:

```suggestion
          jq --indent 4 '."autoload-dev" = {"files": ["vendor/lullabot/drainpipe/scaffold/env/dotenv.php"]}' composer.json | tee composer.new.json
```